cmd/kusk/cmd/root.go

```diff
@@ -49,7 +50,19 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	cobra.CheckErr(rootCmd.Execute())
+	err := rootCmd.Execute()
```

Comment:
can you please revert this back to invoking something like similar to SendAnonymousCMDInfo but pass in the error and rename it. 
You are just repeating what is already done in SendAnonymousCMDInfo

Reply:
Please mark resolved if you're happy with it.